CODE_OF_CONDUCT.md

```diff
@@ -32,6 +32,10 @@ Project maintainers have the right and responsibility to remove, edit, or reject
 
 This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
 
+## Roadmap
+
+Those not affiliated with Confio should not discuss the roadmap.
```

Comment:
There was never any attempt on your part to discuss the roadmap with our team, nor any message asking us about it. We felt we published a decent amount of clarity in February and gladly answered any questions that came up around the Cosmos Hub Proposal 103. This item is untrue and does not represent either current reality or our desired reality

Reply:
I kindly ask you to review [our current standards](https://github.com/CosmWasm/wasmd/blob/main/CODE_OF_CONDUCT.md#our-standards). I do believe your behavior had fallen into the unacceptable areas a number of times, completely unrelated to any recent PRs.

Now that you bring this up, I would like you to reflect on how you could contribute in a way aligned with the current code of conduct. I am happy to discuss this with you in private as well.

Reply:
You write me in DMs many times. If you asked me a simple question there, I could have clarified any doubts easily

Reply:
Thanks @ethanfrey for your response. I had kept this PR intentionally open for some time to let it sink and see if there is any community feedback or opinion that seconds this claim by a different party.

I appreciate feedback from the community and @faddat and Notional have been valuable contributors to the project in 
the past. Nevertheless there are much better ways to participate and make changes than using PRs as platform.

Public channels:
* Regular [community calls](https://vimeo.com/user109497220/videos)
* Public support and discussions on our [discord server](https://chat.cosmwasm.com/)
* Proposals as [Github issues](https://github.com/CosmWasm/wasmd/issues)

As part of proposal [#103](https://www.mintscan.io/cosmos/proposals/103), Confio had shared a roadmap for this year and hosted Twitter spaces to discuss this and answer questions.

As a contributor to the project, @faddat has also a direct channel to me on discord. I have neither received a question or feature requests in DM.

To move on: 
* I see this PR as an expression of @faddat opinion and feelings at the point in time and not more. #1198 was leaving a gab that was not filled. 
* Let us get back to a professional attitude and work on features instead of this